src/Modal/Modal.tsx

```diff
@@ -151,22 +152,35 @@ export const Modal: React.FC<Props> = ({
             as.props.css ? css(as.props.css.styles) : null
           ),
           children: (
-            <div
+            <motion.div
+              animate={{ opacity: 1, scale: 1 }}
+              initial={{ opacity: 0, scale: 0.9 }}
+              transition={{
+                scale: {
+                  type: "spring",
+                  stiffness: 150,
+                  damping: 200,
+                  mass: 0.2,
+                  velocity: 8,
```

Comment:
Q: Where do these values come from?

Reply:
Everything but velocity comes from Adam's codesandbox :)
https://codesandbox.io/s/modal-transition-dknl9 

For some reason the velocity needs to be explicitly set to `8` otherwise the animation looks very different (not enough stretch)

Reply:
Awesome! LGTM